README.md

```diff
@@ -1,7 +1,6 @@
-# ToDoList
+# to-do-list
 
-![](https://img.shields.io/badge/Microverse-blueviolet)
-> A simple to do list.
+> Created a simple to do list.
```

Comment:
- [ ] Kindly add a descriptive description of your project to your README so visitors have a clear idea about your project.